.github/workflows/release.yaml

```diff
@@ -49,20 +49,20 @@ jobs:
         run: go install github.com/swaggo/swag/cmd/swag@latest
 
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
+        uses: docker/setup-qemu-action@v3
 
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
+        uses: docker/setup-buildx-action@v3
 
       - name: Login to GitHub Container Registry
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v4
+        uses: goreleaser/goreleaser-action@v6
```

Comment:
@charlie-haley Am I right in thinking that given `latest`, the repo is already V2 compatible? This was noted in the action breaking changes - https://goreleaser.com/blog/goreleaser-v2/#github-action

Reply:
Yeah I migrated to V2 the other day. It will probably be better to pin a version here instead of `latest` actually